frontend/src/containers/ProfilePage/ProfilePage.js

```diff
@@ -90,7 +90,10 @@ class ProfilePage extends Component {
   }
 
   render() {
+    // console.log(this.props.match);
+
```

Comment:
I started working on trying to get the dropdown to link to the settings tab by using URL parameters. I was following along with something I saw online. But, was struggling and figured I would table it until later. That is why the line above is commented out.